Hoxtygen/news-blog
# setup redux

##  What does this PR do
Set up redux for state management

##  Description of task to be completed
* Install `react-redux`
* Install `@reduxjs/toolkit`
* Install `redux-logger and @types/redux-logger`
* Install `redux-thunk`
* Create store

##  How should this manually be tested
* Clone the repo
* Open a terminal and run `git clone repo_url`
* Cd into the repo and run `npm install`
* Install redux devtool
* Open the devtool to see the created store

##  What are the relevant pivotal tracker stories
[[#174960343](https://www.pivotaltracker.com/story/show/174960343)]

##  Screenshots (If appropriate)


##  Questions (If any):
